test/e2e/testdata/smoketest/mongodb/00_mongodbcluster.yaml

```diff
@@ -6,7 +6,7 @@ metadata:
   name: kb-mongodb-cluster
   namespace: default
   labels:
-    helm.sh/chart: mongodb-cluster-0.6.0-beta.44
+    helm.sh/chart: mongodb-cluster-0.7.0-alpha.0
```

Comment:
suggest helm label to be removed if no other effects.
as this will avoid labels' update in every version update.